packages/react/src/components/Button/Button.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import { useTheme } from '@emotion/react';
 import PropTypes from 'prop-types';
 import useComponentOverrides from '../../theme/useComponentOverrides';
-import styles from './Button.styles';
+import useButtonStyles from './Button.styles';
 
 const getSquareSize = (size) => {
```

Comment:
can we move this to `Button.styles.js` so we don't need to pass it here in `Button`?

Reply:
Sorry Sid, didn't get your question

Reply:
Are you talking about useButtonStyles to move somewhere else ? 
If yes, then that's not possible.. we are exporting our styles from Button.styles.js, but we must have a way to apply it to the elements.. That's why this import is necessary and then use it for the required element

Reply:
we can move this `getSquareSize` function from here to `Button.styles` and use it directly there rather than passing it from Button component

Reply:
Ohh you are talking about getSquareSize, let me see, i think it can be done.. just a sec

Reply:
Done @sidmohanty11